serverless/sql-databases/troubleshooting/failing-backup-restore.mdx

```diff
@@ -0,0 +1,31 @@
+---
+meta:
+  title: Solving failing manual backup and restore operations
+  description: This page lists the different solutions to solve failing backup and restore operations with Serverless SQL Databases.
+content:
+  h1: Solving failing manual backup and restore operations
+  paragraph: This page lists the different solutions to solve failing backup and restore operations with Serverless SQL Databases.
+tags: database serverless-sql-database unable fail backup troubleshooting restore error
+dates:
+  validation: 2024-02-07
+  posted: 2024-02-07
+categories:
+  - serverless
+---
+
+## Manual SQL backup and restore fails
+
+### Problem
+
+When creating or restoring backup manually (i.e. via SQL Administration tools such as PGAdmin, or directly via SQL commands), some errors can happen.
```

Comment:
The capitalization of PGAdmin is not always consistent between the different docs